lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1255,7 +1282,15 @@ def _deploy(
             deployment_operation = deployments.begin_create_or_update(
                 **deployment_parameters
             )
-            wait_operation(deployment_operation, failure_identity="deploy")
+            while True:
```

Comment:
Is there a maximum time on this loop?

Reply:
There is timeout in underlying code.

Reply:
It should be obvious either through code or comments. This looks like it will keep running indefinitely until it fails or returns, there doesn't seem to be a maximum timeout on the loop.